knowledge-base/combobox-virtualization-loader.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Displaying Loading Indicator in ComboBox with Remote Data and Virtualization
+description: Learn how to add a loading indicator in the TelerikComboBox component when using remote data and virtualization functionality in UI for Blazor.
+type: how-to
+page_title: Adding Loader to ComboBox During Remote Data Fetch and Virtualization
+slug: adding-loader-combobox-remote-data-virtualization
```

Comment:
The naming convention is:

componentname-kb-foo-bar

Usually, the `.md` file name is the same, but without the `-kb` part